.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+parser: flow
```

Comment:
Until the upstream react-scripts uses Prettier, we should avoid adding it and having as small of a diff as possible.